src/app/components/nav/header-static/header-static.component.ts

```diff
@@ -102,18 +110,22 @@ export class HeaderStaticComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.updateElements();
     this.innerWidth = window.innerWidth;
-    if (this.innerWidth <= 810) {
-      this.isMenuExpanded = false;
+    if (this.innerWidth > 810) {
+      this.isMenuExpanded = true;
     }
     this.authServiceSubscription = this.authService.change.subscribe((authState) => {
       this.authState = authState;
+      if (this.authState.isLoggedIn) {
+        this.user = this.authState;
+      }
     });
   }
+
   @HostListener('window:resize', ['$event'])
   onResize(event) {
     this.innerWidth = window.innerWidth;
-    if (this.innerWidth <= 810) {
-      this.isMenuExpanded = false;
+    if (this.innerWidth > 810) {
```

Comment:
Create an `expandMenu( )` method to use it in both the places i.e. Line 113 and Line 127 respectively.

Reply:
okay